package.json

```diff
@@ -33,9 +33,20 @@
     "build:umd:min": "cross-env BABEL_ENV=rollup NODE_ENV=production rollup -c -o dist/react-redux.min.js",
     "build": "npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd:min",
     "clean": "rimraf lib dist es coverage",
-    "lint": "eslint src test",
+    "lint": "eslint src test/utils test/components test/getTestDeps.js",
     "prepare": "npm run clean && npm run build",
+    "pretest": "node ./install-test-deps.js",
     "test": "jest",
+    "test:ci": "node ./run-tests.js",
+    "pretest:watch": "node ./install-test-deps.js",
+    "test:watch": "jest --watch",
+    "test:14": "cross-env REACT=0.14 node ./run-tests.js",
+    "test:15": "cross-env REACT=15 node ./run-tests.js",
+    "test:16.2": "cross-env REACT=16.2 node ./run-tests.js",
+    "test:16.3": "cross-env REACT=16.3 node ./run-tests.js",
+    "test:16.4": "cross-env REACT=16.4 node ./run-tests.js",
+    "test:all": "npm run test:14 && npm run test:15 && npm run test:16.2 && npm run test:16.3 && npm run test:16.4",
```

Comment:
Could we add a special `REACT=all` case here instead? That would remove the need to maintain these scripts over time.